projects/canopy-test-app/src/app/app.component.html

```diff
@@ -291,11 +291,20 @@
               <lg-hint>e.g. 05 12 1980</lg-hint>
             </lg-date-field>
 
+            <lg-checkbox-group formControlName="colors">
+              Test checkbox group
+              <lg-hint>This is a standard checkbox group</lg-hint>
+              <lg-toggle value="red">Red</lg-toggle>
```

Comment:
They do look nicer and initially I wanted to build them like this, that would imply just adding more selectors to the toggle component and also take the variant based on that, but then I thought it would be a bit overkill to also have `lg-toggle` with variant prop and a selector for variant. 
If we choose to have both implementations, I don't think there will be any big compromise, and we could avoid a breaking change. But there is also the question if I should take the variant from the group at all if we implement it like this? 
We will have 3 ways of setting the variant on the toggle component and that's the part that is a bit overkill from my point a view, but I am ok on doing that and opened to discussions on this one.

Reply:
Done. The variant gets set as follows:
- If the user uses a group, the toggle will take the variant of the group.
- If the user uses a toggle by itself, with the selector, the variant is set based on the selector.
- If the user uses a toggle by itself, with the variant `@Input` prop, the variant is set based on users variant input.

Reply:
Nice one! excellent stuff.